webapp/home/templates/home/snippets/about/about.html

```diff
@@ -40,11 +40,11 @@ <h3 id="operational-partners">
 </h3>
 
 <p>
-  The Australian BioCommons (hosted by
+Australian BioCommons (hosted by
   <a href="https://www.unimelb.edu.au/" target="_blank">The University of Melbourne</a>
-  ) operates Galaxy Australia in collaboration with
+) operates Galaxy Australia in collaboration with
```

Comment:
Unwanted space around tags can be tedious. You'll need to take the bracket right up to the closing tag to get rid of this one: `</a>)`